plugins/MauticFocusBundle/Controller/AjaxController.php

```diff
@@ -40,4 +41,91 @@ public function generatePreviewAction(Request $request): JsonResponse
 
         return $this->sendJsonResponse($responseContent);
     }
+
+    public function getViewsCountAction(Request $request, CacheProvider $cacheProvider): JsonResponse
```

Comment:
Could you please trigger these 2 new actions with a functional test? It seems they are not being run in PHPUNIT.

Reply:
Parent class requires `MauticFactory`, which is deprecated (I'm not using it). I need to override the constructor to inject the `CacheProvider`. Do you know better workaround than just ignore this constructor?

Reply:
You are fast! Thanks!